portal/src/views/projects/ProjectBigMap.vue

```diff
@@ -111,6 +121,17 @@ export default Vue.extend({
         exploreContext(): ExploreContext {
             return new ExploreContext(this.project.id);
         },
+        levels(): object[]{
```

Comment:
I would return from inside the try and then return an empty list from the catch or outside the try/catch.